src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -46,31 +48,23 @@ public RepoConfigCsvParser(Path csvFilePath) throws IOException {
      * {@code branch}.
      */
     @Override
-    protected void processLine(List<RepoConfiguration> results, String[] elements) throws InvalidLocationException {
-        RepoLocation location = new RepoLocation(getValueInElement(elements, LOCATION_POSITION));
-        String branch = getValueInElement(elements, BRANCH_POSITION, RepoConfiguration.DEFAULT_BRANCH);
+    protected void processLine(List<RepoConfiguration> results, CSVRecord record) throws InvalidLocationException {
+        RepoLocation location = new RepoLocation(get(record, LOCATION_POSITION));
+        String branch = getOrDefault(record, BRANCH_POSITION, RepoConfiguration.DEFAULT_BRANCH);
 
-        boolean isFormatsOverriding = isElementOverridingStandaloneConfig(elements, FILE_FORMATS_POSITION);
-        if (isFormatsOverriding) {
-            removeOverrideKeywordFromElement(elements, FILE_FORMATS_POSITION);
-        }
-        List<Format> formats = Format.convertStringsToFormats(getManyValueInElement(elements, FILE_FORMATS_POSITION));
+        boolean isFormatsOverriding = isElementOverridingStandaloneConfig(record, FILE_FORMATS_POSITION);
+        List<Format> formats = Format.convertStringsToFormats(
+                getAsListWithoutOverridePrefix(record, FILE_FORMATS_POSITION));
 
-        boolean isIgnoreGlobListOverriding = isElementOverridingStandaloneConfig(elements, IGNORE_GLOB_LIST_POSITION);
-        if (isIgnoreGlobListOverriding) {
-            removeOverrideKeywordFromElement(elements, IGNORE_GLOB_LIST_POSITION);
-        }
-        List<String> ignoreGlobList = getManyValueInElement(elements, IGNORE_GLOB_LIST_POSITION);
+        boolean isIgnoreGlobListOverriding = isElementOverridingStandaloneConfig(record, IGNORE_GLOB_LIST_POSITION);
```

Comment:
booleans not in use anymore?

Reply:
It's still being used in an input parameter upon initializing `RepoConfiguration` 

https://github.com/reposense/RepoSense/blob/4c915c422e30b20f8c1674c036dc2c7a984af162/src/main/java/reposense/parser/RepoConfigCsvParser.java#L74-L77